scipy/optimize/cobyla.py

```diff
@@ -165,13 +169,14 @@ def fmin_cobyla(func, x0, cons, args=(), consargs=None, rhobeg=1.0,
 
     # build constraints
     con = tuple({'type': 'ineq', 'fun': c, 'args': consargs} for c in cons)
-
+    
```

Comment:
please remove this errant whitespace, is it a tab?

Reply:
It was just 4 spaces left there by mistake. They are gone now.